nodes/list_masks/mask_join.py

```diff
@@ -18,14 +18,14 @@
 
 from itertools import cycle
 import bpy
-from bpy.props import BoolProperty, IntProperty, StringProperty
+from bpy.props import BoolProperty, IntProperty, StringProperty, EnumProperty
 from sverchok.node_tree import SverchCustomTreeNode
-from sverchok.data_structure import changable_sockets, updateNode
+from sverchok.data_structure import changable_sockets, updateNode, list_match_modes, list_match_func
 
 
-class SvMaskJoinNode(bpy.types.Node, SverchCustomTreeNode):
+class SvMaskJoinNodeMK2(bpy.types.Node, SverchCustomTreeNode):
     '''Mask Join'''
-    bl_idname = 'SvMaskJoinNode'
+    bl_idname = 'SvMaskJoinNodeMK2'
```

Comment:
The new name also should be fixed in the index.md file

Reply:
That was not said in the link you gave me.

Reply:
Should be added then.